apps/src/templates/teacherHomepage/CollapsibleSection.jsx

```diff
@@ -72,13 +72,24 @@ const CollapsibleSection = React.createClass({
   renderContent() {
     if (this.state.open) {
       return (
-        <div style={styles.content}>
+        <div>
           {React.Children.map(this.props.children, (child, index) => {
-            return (
-              <div key={index}>
-                {child}
-              </div>
-            );
+            if (index % 2 === 0) {
+              return (
+                <div key={index}>
+                  {child}
+                    <div style={styles.spacer}>
+                      .
+                    </div>
+                </div>
+              );
+            } else {
+              return (
+                <div key={index}>
+                  {child}
+                </div>
+              );
+            }
```

Comment:
Could simplify this by having just `{(index % 2 === 0) && <div style={styles.spacer}>.</div>}`